packages/compiler-cli/src/ngtsc/annotations/src/component.ts

```diff
@@ -338,6 +339,15 @@ export class ComponentDecoratorHandler implements
 
     // Next, read the `@Component`-specific fields.
     const {decorator: component, metadata, inputs, outputs} = directiveResult;
+    const encapsulation: number =
+        this._resolveEnumValue(component, 'encapsulation', 'ViewEncapsulation') || 0;
```

Comment:
Can we use enum value here instead of an integer?
```suggestion
        this._resolveEnumValue(component, 'encapsulation', 'ViewEncapsulation') || ViewEncapsulation.Emulated;
```
as an option, we can also put in as a comment:
```suggestion
        this._resolveEnumValue(component, 'encapsulation', 'ViewEncapsulation') || 0 /* ViewEncapsulation.Emulated */;
```
Also since the value might be `0`, we can probably use `??` instead of `||`?